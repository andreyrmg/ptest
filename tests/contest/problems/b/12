90 3
83 39 31
33 81 15 25 20 26 59 4 6 7 8 11 19 22 27 28 29 37 45 49 57 60 65 68 70 72 73 75 79 83 85 86 89 90
29 23 41 26 72 8 10 16 28 29 31 37 40 44 54 58 59 60 61 64 70 71 81 82 84 85 87 88 89 90
19 86 5 11 25 28 30 40 44 46 48 49 65 67 73 79 82 83 85 88
49 60 42 15 84 19 67 73 71 47 39 10 1 66 37 85 72 38 5 25 81 43 44 32 36 17 57 56 34 11 22 78 41 23 40 82 8 24 29 30 45 49 55 59 65 70 74 75 88 89
38 4 3 76 45 85 7 60 88 14 72 53 83 55 29 90 73 54 63 24 74 12 61 43 80 18 8 11 23 34 40 48 51 57 58 59 68 71 89
50 79 54 48 10 1 61 50 57 66 40 26 84 85 46 17 14 20 75 35 77 63 51 18 89 34 25 82 38 64 12 58 62 8 13 33 37 11 16 19 22 31 36 43 49 60 67 68 71 81 90
47 5 9 38 49 12 74 90 78 88 55 73 31 18 30 84 77 22 81 8 64 17 1 37 27 76 52 60 63 66 72 39 19 54 13 28 29 44 59 61 65 67 70 71 79 85 86 87
43 6 7 69 19 60 72 15 30 2 13 82 52 49 32 5 62 40 31 22 51 90 86 70 1 46 78 10 55 4 36 54 11 24 29 56 59 67 68 73 74 75 85 88
23 7 18 22 27 29 30 34 40 44 46 48 58 67 68 70 71 73 74 79 82 86 88 90
38 4 6 8 77 83 58 39 90 46 80 35 52 14 48 69 44 2 59 75 70 55 54 47 82 18 25 31 37 43 45 51 60 61 64 73 86 87 88
46 4 85 30 79 87 6 38 66 70 77 33 90 58 39 35 18 12 61 84 49 75 63 5 1 60 34 8 28 48 3 21 20 24 47 51 17 31 37 40 45 55 64 65 68 74 88
22 5 6 7 11 19 20 21 25 30 31 43 49 55 58 61 68 74 81 83 86 87 90
25 6 8 7 38 16 20 21 24 28 31 33 37 45 54 55 58 64 70 73 78 79 84 86 87 88
28 5 6 10 31 16 18 20 28 33 37 40 43 44 55 58 62 64 65 68 70 72 73 77 78 79 82 86 89
41 1 4 8 47 21 50 68 37 83 65 62 88 60 29 56 51 53 81 67 77 55 16 18 22 23 24 25 30 31 33 34 40 43 45 46 49 58 61 70 86 87
46 15 75 76 81 40 47 43 86 21 13 20 34 61 14 64 27 58 55 54 50 73 52 42 68 6 78 23 69 25 18 70 45 2 74 79 33 28 32 30 87 29 85 26 37 44 83
31 4 6 7 86 83 61 37 89 62 87 28 65 88 84 85 68 74 11 40 18 19 21 24 27 29 43 45 49 75 77 90
54 5 6 7 11 16 21 81 59 87 72 26 46 51 64 32 56 9 42 50 31 66 69 15 85 48 75 88 62 63 23 27 10 53 86 78 17 14 19 34 41 33 30 40 43 49 55 58 60 70 73 74 83 89 90
37 4 7 8 18 87 17 83 6 86 42 21 23 50 33 72 12 51 70 78 90 1 60 25 29 30 37 43 58 59 61 64 65 68 73 74 80 89
45 1 6 11 16 35 76 55 13 41 14 72 42 39 50 80 12 21 46 62 69 78 56 60 37 25 47 64 83 26 28 33 40 44 49 58 59 61 65 67 68 73 74 79 85 89
31 11 15 16 18 19 20 60 57 12 49 46 80 53 47 27 48 77 17 40 56 87 63 13 81 24 25 28 43 44 70 78
34 4 7 8 1 75 76 23 72 77 55 9 15 6 45 49 89 30 58 27 28 29 43 56 67 68 71 74 78 82 84 85 87 88 90
23 2 4 16 18 19 22 88 45 38 70 26 47 49 5 15 86 24 25 41 55 64 71 90
42 5 11 34 13 38 30 86 43 60 68 15 21 74 81 44 29 84 73 71 76 23 47 82 4 33 66 46 17 72 83 8 56 64 63 28 31 40 51 55 58 61 90
45 1 4 6 16 20 19 72 21 85 41 82 36 58 73 56 40 46 78 45 47 79 23 12 10 43 31 15 3 68 66 37 29 28 32 38 42 49 65 67 70 75 77 83 88 89
28 1 2 6 16 18 20 23 49 58 89 59 64 84 34 29 42 43 56 60 61 62 68 71 73 75 78 79 83
35 7 16 18 21 87 76 59 55 40 22 1 90 33 56 31 81 61 73 79 9 88 44 17 48 30 43 51 64 65 68 71 74 78 82 89
47 11 16 17 22 24 44 63 87 83 38 55 68 70 42 13 40 69 62 20 3 7 88 74 48 2 65 25 46 21 57 51 14 1 54 36 31 37 43 45 58 64 66 67 71 77 84 86
52 5 15 16 24 25 45 34 79 2 50 37 33 8 7 88 85 62 31 69 82 90 9 32 17 19 1 75 43 40 4 26 73 54 83 35 86 77 38 22 44 30 42 55 58 59 64 67 70 71 74 78 89
46 7 8 11 16 22 24 27 42 3 63 89 12 64 86 54 19 35 56 53 4 61 9 85 50 34 78 18 37 29 67 74 15 31 33 38 49 55 65 68 70 75 76 79 84 88 90
42 7 8 14 18 25 27 29 59 28 66 65 45 15 10 64 89 67 53 12 2 54 6 84 11 36 88 24 41 46 52 13 40 30 86 44 43 49 51 58 77 78 90
17 4 8 16 18 29 25 49 34 40 51 56 60 68 74 85 87 90
37 6 11 16 18 19 24 27 29 71 55 65 66 14 13 59 40 39 74 44 83 64 15 30 84 63 20 34 36 43 51 60 61 67 68 76 87 90
31 4 6 11 16 18 24 26 29 30 9 86 5 35 15 54 57 79 32 84 33 71 42 37 40 43 48 55 68 73 85 88
19 6 10 11 20 29 30 34 46 37 40 43 55 65 69 71 73 78 82 86
27 4 8 25 28 31 42 33 54 75 6 83 46 37 44 60 64 67 68 71 74 78 79 80 84 86 88 90
53 4 6 7 15 17 20 25 29 30 68 14 82 44 19 81 10 11 13 86 75 36 69 28 90 71 34 1 51 50 35 89 43 87 66 57 49 59 46 41 77 16 48 62 2 61 38 40 58 60 64 67 70 73
21 4 6 7 11 13 23 24 28 29 62 30 25 37 40 45 46 51 73 74 81 89
18 4 7 10 11 20 33 45 55 57 64 70 74 75 78 83 85 89 90
55 4 6 8 16 17 21 25 27 28 29 31 33 48 78 42 68 60 14 47 20 35 2 15 9 11 72 76 67 81 38 73 5 34 3 49 66 63 37 70 18 88 24 32 59 80 83 46 43 55 56 58 71 79 84 86
26 2 4 18 20 25 31 37 42 43 86 87 64 46 23 61 74 70 45 51 55 59 60 67 84 85 89
30 4 16 18 19 20 28 30 34 36 40 41 74 80 29 44 26 25 43 90 78 71 55 58 65 67 68 70 76 82 84
48 4 5 16 24 25 27 29 37 41 42 18 12 74 15 53 21 19 22 6 85 64 54 40 71 49 47 28 55 14 34 31 35 17 61 51 70 26 33 10 76 75 59 79 82 60 67 86 90
45 4 10 24 27 28 29 31 33 37 42 88 73 7 80 81 9 14 67 36 65 3 16 48 50 2 72 84 63 20 52 76 21 45 49 55 59 70 71 74 77 78 85 86 89 90
42 5 16 22 23 25 29 31 1 78 4 52 71 59 46 15 41 81 28 13 55 39 47 17 44 38 88 68 10 48 11 84 58 60 49 65 67 70 74 77 79 86 89
27 6 8 10 18 20 21 24 25 28 31 35 36 37 40 41 45 76 38 3 9 15 90 60 70 73 78 87
28 4 10 11 15 16 20 21 23 24 25 40 43 45 51 58 60 65 67 68 70 72 75 77 79 86 87 88 90
31 6 10 11 18 21 27 28 37 40 44 45 62 78 34 76 63 58 9 87 84 3 5 57 59 67 74 77 79 85 88 89
44 7 8 11 21 22 23 26 32 37 40 43 17 25 12 70 31 6 3 20 65 15 45 53 18 4 69 63 68 30 1 44 62 76 66 77 79 60 75 81 82 83 85 86 88
25 6 15 16 18 19 20 29 30 37 44 68 51 60 65 70 71 73 75 77 78 79 83 84 88 90
41 6 8 11 15 18 19 28 37 43 33 41 72 5 47 78 52 27 10 63 31 50 56 24 59 88 38 86 69 32 55 58 65 67 68 74 79 81 82 83 85 89
26 7 8 10 16 31 44 45 51 55 59 67 68 69 70 71 73 74 75 76 78 79 83 84 85 89 90
16 5 15 18 21 30 31 43 49 86 55 58 67 69 74 82 85
33 5 6 7 8 10 16 28 29 30 31 34 36 43 72 13 58 78 90 2 56 59 60 67 68 71 73 75 77 79 81 85 86 88
58 5 7 8 10 15 16 20 22 27 28 33 43 45 52 81 71 72 24 51 69 79 4 40 35 80 30 39 57 61 29 44 42 34 84 59 18 74 63 12 41 68 53 14 66 86 23 64 13 60 11 56 67 70 73 78 85 88 89
31 4 15 18 20 21 24 25 27 30 51 40 54 84 86 55 32 75 22 78 59 76 26 87 8 60 68 69 80 83 88 89
25 4 6 21 28 34 37 55 1 39 48 5 77 63 58 60 62 65 67 70 74 75 78 85 88 89
48 6 10 11 16 22 25 26 45 48 54 24 69 89 5 47 85 68 18 82 13 40 42 71 60 19 12 2 83 29 9 63 14 15 75 28 57 87 80 53 73 31 81 37 79 20 77 51 64
42 1 10 18 26 27 31 33 37 40 43 45 51 55 56 62 80 48 7 29 20 5 41 44 90 83 85 19 8 4 89 68 52 54 2 61 64 70 74 79 86 87 88
48 4 5 7 8 11 15 19 20 21 24 40 43 45 55 58 37 32 49 71 63 50 68 69 2 46 33 26 78 1 84 18 90 81 41 56 54 62 73 57 88 47 10 6 36 89 70 74 77
33 5 6 11 16 17 27 30 37 41 43 55 59 7 62 15 12 85 90 26 84 33 24 2 10 83 19 68 20 65 73 75 79 82
28 6 8 15 17 18 20 28 29 37 38 48 49 59 60 61 57 88 14 26 68 70 73 74 75 77 84 87 89
28 5 6 7 11 18 21 24 28 30 33 40 44 48 49 51 55 57 58 60 76 67 69 73 74 79 83 86 89
40 6 7 16 18 20 24 26 30 31 33 41 43 55 59 71 70 2 11 37 76 13 23 80 39 29 28 72 36 27 10 86 19 58 14 67 69 73 77 85 88
38 15 17 28 31 33 44 49 47 3 81 57 73 20 14 7 4 27 42 72 11 19 78 25 85 45 69 1 51 30 75 35 86 61 50 77 87 88 90
22 4 6 7 11 18 24 25 31 33 37 40 49 55 77 28 68 70 74 78 86 88 90
46 4 15 30 31 40 44 20 53 22 64 42 76 7 6 25 36 71 48 28 43 51 78 37 54 55 8 72 41 52 45 57 9 82 63 83 74 73 47 89 3 33 69 29 75 79 90
55 15 16 17 24 25 28 37 40 45 49 50 55 58 59 60 61 34 89 9 8 6 87 56 42 78 11 19 22 74 33 54 12 5 75 62 30 27 86 51 1 32 14 20 79 80 52 47 70 36 66 26 71 77 84 90
29 8 10 16 18 20 28 29 37 49 51 55 58 60 65 67 56 53 63 64 52 35 89 77 74 76 81 82 85 88
53 8 10 11 16 19 23 28 40 41 43 49 64 68 25 9 15 44 42 47 89 30 4 85 57 62 13 39 84 7 71 45 66 52 55 90 59 60 81 21 29 77 2 37 1 74 46 14 18 50 73 75 78 82
46 4 24 33 34 37 42 43 45 55 58 60 64 67 70 23 68 7 40 73 5 29 54 75 50 35 52 86 2 27 9 81 22 36 80 28 89 44 79 26 6 77 78 84 85 87 88
27 2 4 5 7 8 18 19 20 22 24 25 40 44 51 54 55 64 65 67 1 87 14 47 73 74 79 89
54 4 5 7 16 24 25 27 29 40 44 58 60 65 67 71 52 20 72 63 62 18 34 35 10 76 38 89 77 19 90 37 61 50 9 64 1 3 70 55 84 8 46 14 78 54 26 79 83 13 74 75 81 82 86
55 5 7 16 17 24 28 30 33 41 42 43 55 67 68 70 39 84 63 51 20 9 38 85 73 44 45 80 22 57 11 48 12 52 4 53 66 19 72 8 29 59 69 36 18 27 60 87 32 81 62 75 77 78 79 82
45 6 10 11 16 18 22 29 36 37 43 56 58 65 68 71 30 61 4 62 85 39 17 50 47 1 49 70 86 52 57 80 26 89 87 8 73 74 25 88 67 54 78 83 84 90
30 5 7 16 20 22 24 27 40 43 44 46 48 49 56 63 64 67 73 33 52 89 30 42 80 84 69 77 79 81 86
39 6 7 10 11 15 21 22 29 37 49 57 58 66 69 70 73 76 62 50 31 44 60 82 45 54 74 65 25 47 88 14 68 71 48 64 28 17 80 83
52 4 7 8 16 18 19 20 25 30 40 42 45 48 51 54 56 60 65 67 68 73 79 90 36 52 14 82 21 39 85 46 13 66 55 70 22 27 57 80 44 81 31 84 74 26 71 50 75 35 29 88 89
44 6 11 16 25 27 29 34 43 49 55 58 68 71 73 78 36 51 14 52 54 59 72 3 74 47 45 48 26 30 86 13 80 7 76 87 20 9 61 84 1 63 67 40 50
25 5 10 20 21 40 42 44 55 58 59 64 68 71 74 75 76 78 79 77 36 19 56 83 85 90
34 1 4 7 15 16 18 21 24 27 37 40 44 45 55 58 60 65 70 71 74 78 54 2 51 69 90 76 12 86 6 73 49 38 88
33 4 6 8 10 24 25 29 37 43 58 67 77 78 22 84 90 53 73 74 14 27 3 69 49 42 9 51 35 86 2 70 83 61
39 5 10 15 17 19 20 24 28 29 33 36 40 58 59 61 67 73 82 88 3 52 63 49 26 77 75 50 1 16 51 25 39 80 56 18 84 12 89 90
45 4 6 7 11 17 24 26 31 33 34 44 45 48 55 56 60 61 70 73 74 76 78 79 82 83 50 68 41 40 71 28 62 36 2 42 30 75 52 22 13 85 86 87 88 90
46 4 5 6 11 16 17 18 25 29 30 43 58 59 61 65 70 74 75 78 53 22 80 52 49 44 8 51 55 1 90 34 3 57 20 48 54 2 84 32 7 39 64 69 41 71 89
51 3 8 16 17 18 19 23 24 29 30 31 34 37 41 51 53 55 56 64 65 68 71 75 79 81 82 43 35 14 28 15 13 47 63 44 7 1 73 40 36 12 49 45 9 66 10 54 76 59 84 90
35 11 16 17 18 19 21 27 28 37 41 48 56 58 68 72 74 75 79 7 22 84 62 2 32 71 13 47 65 59 33 46 15 10 12 90
50 5 7 15 17 18 23 27 28 29 31 40 44 45 51 60 62 75 77 83 55 34 64 9 8 54 66 57 59 13 69 78 49 4 50 11 10 48 84 65 81 2 47 3 36 30 71 56 25 22 90
46 6 17 22 26 30 31 37 58 59 60 67 68 69 70 71 73 75 76 39 27 38 83 4 20 63 56 78 48 18 62 29 19 14 55 72 25 51 41 5 1 2 44 52 45 57 85
53 5 7 8 10 11 19 27 29 37 42 46 54 59 60 61 70 73 78 81 82 85 33 32 30 1 2 50 52 17 80 68 83 47 9 84 22 23 75 66 88 39 43 87 86 36 12 65 6 67 18 44 31 24
