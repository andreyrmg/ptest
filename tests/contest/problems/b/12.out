61 12 17 